### PR TITLE
Cleanups: Dangling pointers, unused variables, signness

### PR DESCRIPTION
**Pull request by gerstrong**
Please do not forget to update the mmex.pot file and write information about the fixed bug in the prerelease [page](https://github.com/moneymanagerex/moneymanagerex/releases).

- - -
This change is [Reviewable](https://reviewable.io/reviews/moneymanagerex/moneymanagerex/6939)

### CODE REVIEW

File: src/import_export/univcsvdialog.cpp [hunk 1]
```diff
@@ -198,7 +198,7 @@ void mmUnivCSVDialog::CreateControls()
     m_text_ctrl_->Connect(ID_FILE_NAME
         , wxEVT_COMMAND_TEXT_ENTER, wxCommandEventHandler(mmUnivCSVDialog::OnFileNameEntered), nullptr, this);
 
-    const wxString& file_button_label = _("&Browse...");
+    const auto file_button_label = wxString("&Browse...");
```
Comment (vomikan) [inline src/import_export/univcsvdialog.cpp:201]: why?

File: src/mmSimpleDialogs.cpp [hunk 2]
```diff
@@ -1017,7 +1017,7 @@ void mmErrorDialogs::InvalidFile(wxWindow *object, bool open)
 
 void mmErrorDialogs::InvalidAccount(wxWindow *object, bool transfer, TOOL_TIP tm)
 {
-    const wxString& errorHeader = _("Invalid Account");
+    const auto errorHeader = wxString("Invalid Account");
```
Comment (vomikan) [inline src/mmSimpleDialogs.cpp:1020]: Should be  wxGetTranslation("Invalid Account") ?
or just "Invalid Account"

File: src/import_export/univcsvdialog.cpp (hunk 1 above)
Comment (whalley) [reply src/import_export/univcsvdialog.cpp:201]: Good point, I don't see an issue with... `const wxString file_button_label = _("&Browse..."); `

File: src/mmSimpleDialogs.cpp (hunk 2 above)
Comment (whalley) [reply src/mmSimpleDialogs.cpp:1020]: Good point, I don't see an issue with... `const wxString errorHeader = _("Invalid Account"); `

**Pull request merged**